app/assets/stylesheets/bulma_and_buefy.scss

```diff
@@ -1,9 +1,18 @@
 // Import Bulma's core if needed for customizations
-// @import "bulma/sass/utilities/_all";
+@import "bulma/sass/utilities/_all";
+@import "bulma/sass/components/navbar";
 
 // Our customizations
 $input-placeholder-color: hsl(0, 0%, 86%); //has-text-grey-lighter
 
+.subnav {
+  .navbar-item {
+    &.is-active {
+      color: $navbar-item-hover-color;
```

Comment:
By default, the navbar (when active) remains `#000`. This ensures that anchor tags in a `subnav` which are active are visible.